### DISCUSSION TITLE
阅读Milvus代码，有什么好的建议吗？

### DISCUSSION THREAD

**Discussion by gobigrassland** [milvus-io/milvus#17561] [Q&A and General discussion]
看了Zilliz公众号上介绍，主要包括Deep Dive系列。在实际使用Milvus过程中遇到一些问题，需要阅读Milvus代码才能深刻理解，但是刚接触Milvus，代码目前看起来有些吃力，大家有什么好的建议吗？

**Comment by yhmo** [comment]
也没什么捷径，就是多看。说实话我看着也费劲...
